python-requirements.txt

```diff
@@ -12,7 +12,7 @@ pycryptodome==3.15.0
 pyelftools==0.29
 pytest==7.0.1
 pytest-timeout==2.1.0
-rich==12.5.1
+rich==13.3.5
```

Comment:
@timothytrippel would this break airgapped builds?

Reply:
Potentially, but if I update something first then it won't. Let me submit an update to an internal infra to take of this and comment back here when I am done. One moment.

Reply:
Thanks @timothytrippel !

Reply:
Should be Ok to merge now

Reply:
Thanks a lot!

Reply:
It turns out this is actually not compatible, I created a PR to update the newest version that is compatible with the CENTOS7 machines running the nightlies: https://opentitan-review.googlesource.com/c/asic_top/+/15220

Will this be an issue?